server.js

```diff
@@ -17,10 +18,69 @@ app.use(cors());
 app.use(express.json());
 
 // Start defining your routes here
+
+////////////////
+const ThoughtsSchema = new mongoose.Schema({
+  message: {
+    type: String,
+    required: true,
+    minlength: 1,
+    maxlength: 140,
+    trim: true
```

Comment:
trim:true, I guess this makes the message 140 letters if it's too long. Good idea! :)